404/model.html

```diff
@@ -0,0 +1,525 @@
+                    <div class="box">
+                      <a href="#">Family</a>
+                      <a href="#">Architecture</a>
+                      <a href="#">Moddel</a>
+                      <a href="#">Wedding</a>
+                      <a href="#">Events</a>
+                    </div>
+                  </div>
+                </div>
+                <a href="#" target="_blank">
+                  <i class=" fab fa-facebook-f sh-header-builder-main-element-icon"></i>
+                </a>
+                <a href="#" target="_blank">
+                  <i class=" fab fa-twitter sh-header-builder-main-element-icon"></i>
+                </a>
+                <a href="#">
+                  <i class="fa fa-search sh-header-builder-main-element-icon"></i>
+                </a>
+                <a href="#">
+                  <button type="button" class="btn btn-primary btn-lg">Contact me</button>
+                </a>
+              </div>
+              <div class="dropdown container-position">
+                <a class="btn btn-secondary menu-button " href="#" role="button" id="dropdownMenuLink"
+                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  <div></div>
+                  <div></div>
+                  <div></div>
+                </a>
+    
+                <div class="dropdown-menu dropcontent box" aria-labelledby="dropdownMenuLink">
+                  <a href="#">Home</a>
+                  <a href="#">Family</a>
+                  <a href="#">Architecture</a>
+                  <a href="#">Moddel</a>
+                  <a href="#">Wedding</a>
+                  <a href="#">Events</a>
+                </div>
+              </div>
+    
+    
+            </div>
+           
+        </div>
+        </div>
+        </nav>
+      </header>
+
+
+
+    <!-- Sara end -->
+
+    <!-- Qays -->
+    <div>
+        <div class="swiper-container">
+            <div class="swiper-slide ">
+                <img src="./imgs/female_model/1.jpg" alt="female_model_1">
+                <img src="./imgs/female_model/2.jpg" alt="female_model_2">
+                <img src="./imgs/female_model/3.jpg" alt="female_model_3">
+                <img src="./imgs/female_model/4.jpg" alt="female_model_4">
+                <img src="./imgs/female_model/5.jpg" alt="female_model_5">
+                <img src="./imgs/female_model/6.jpg" alt="female_model_6">
+            </div>
+        </div>
+    </div>
+    <div class="container mb-5">
+        <div class="row">
+            <div class="col-lg-8 col-md-12 ">
+                <div>
+                    <a class="btn btn-light btn-lg rounded-circle mr-3 mb-3 mt-5 ">
+                        <i class="fas fa-arrow-left"></i>
+                    </a>
+                    <a class="btn btn-light btn-lg rounded-circle mr-3 mb-3 mt-5">
+                        <i class="fas fa-arrow-right"></i>
+                    </a>
```

Comment:
Great job on using the bootstrap classes here to make the arrow buttons. As you can see with just adding ready classes to the element you can achieve the rounded buttons in an easy and quick way!